R/get_ss3_exe.r

```diff
@@ -0,0 +1,85 @@
+#' Download the Stock Synthesis (SS3) executable
+#'
+#' Downloads the SS3 executable according to specified version and the user 
+#' operating system.
+#'
+#' @param dir The directory that you would like the executable downloaded to.
+#' @param version A character string of the executable version tag to download 
+#' (e.g.'v3.30.20' or 'v3.30.18').
```

Comment:
Maybe expand `version` to add a sentence like. "A list of tags is available at https://github.com/nmfs-stock-synthesis/stock-synthesis/tags."